frontend/composables/recipes/use-recipe-nutrition.ts

```diff
@@ -0,0 +1,47 @@
+import { useContext } from "@nuxtjs/composition-api";
+
+
+type NutritionLabelType = {
```

Comment:
can we use an interface here, i think this matches better what we used in the rest of the use-x files. 

```suggestion
export interface NutritionLabelType {
```